docs/source/usage/plugins/checkpoint.rst

```diff
@@ -21,18 +21,17 @@ What is the format of the created files?
 
 We write our fields and particles in an open markup called :ref:`openPMD <pp-openPMD>`.
 
-For further details, see the according sections in :ref:`the openPMD API <usage-plugins-openPMD>` and :ref:`ADIOS <usage-plugins-ADIOS>`.
+For further details, see the according section in :ref:`the openPMD API <usage-plugins-openPMD>`.
```

Comment:
Should be "For further details, see the :ref:`the openPMD API <usage-plugins-openPMD>` section".